lib/src/constants/constants_color.dart

```diff
@@ -0,0 +1,11 @@
+class AppColorConstants {
+  static const int mainGreenColor = 0xff30BE76;
```

Comment:
what differents between mainGreenColor and systemGreenColor and systemOrangeColor

Reply:
![image](https://user-images.githubusercontent.com/33689489/127884548-cddf1187-c9a9-4228-8fb9-3504eeb6acc1.png)
My team choose each name color in global constants

Reply:
got it, tks